src/magnify/apps/core/admin.py

```diff
@@ -1,7 +1,7 @@
 """Admin classes and registrations for Magnify's core app."""
 from django.contrib import admin
 
-from .models import Room, ResourceAccess, User
+from .models import ResourceAccess, Room, User
 
 
```

Comment:
Not sure about the purpose of the commit `9876257f90e8e9435021ebc061de82516c685a7a` by @NathanVss  ? ( @sampaccoud )

Reply:
fix when squash